### PR TITLE
Revise slide image captions

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- move image caption meta to CMB
- add related Work post search field
- add caption updating to slider switch event

### CODE REVIEW

File: lib/meta-boxes.php [hunk 1]
```diff
@@ -236,5 +236,59 @@ function igv_cmb_metaboxes() {
     ),
   ) );
 
+
+  // ATTACHMENTS
+
+  $attachment_metabox = new_cmb2_box( array(
```
Comment (varffs) [inline lib/meta-boxes.php:242]: These dont seem to show up anywhere I can find in my local though the codebase looks correct. Where should I be looking?

Comment (mikeyrayvon) [reply lib/meta-boxes.php:242]: ooops sorry. should have xplained.

Media > Click an attachment > Scroll down > Edit more details

**Review by varffs — COMMENTED** [review]
It seems a bit extreme that you have to set the caption each time for each post type. Perhaps it should fallback to check for the default if not set. But works as the code expects

**Comment by mikeyrayvon** [comment]
ah fuck yea thats what the default was for. i forgot to write the fallback. i'll make an issue

**Pull request merged**